backend/controllers/viewingStatusController.ts

```diff
@@ -0,0 +1,50 @@
+import express from 'express'
+
+import { ViewingStatusApplicationService } from '../application/viewingStatus'
+
+export class ViewingStatusController {
+  updateViewingStatus = async (req: express.Request, res: express.Response) => {
+    try {
+      const { isWatched, userId, videoId } = req.body
+      const viewingStatus = await ViewingStatusApplicationService.update({
+        isWatched,
+        userId,
+        videoId,
+      })
+      res.status(201).json(viewingStatus)
```

Comment:
```suggestion
      res.status(200).json(viewingStatus)
```
にしておきましょうか

https://www.seohacks.net/blog/1147/#:~:text=%E3%81%AA%E3%81%84%E3%81%A7%E3%81%97%E3%82%87%E3%81%86%E3%80%82-,HTTP%E3%82%B9%E3%83%86%E3%83%BC%E3%82%BF%E3%82%B9%E3%82%B3%E3%83%BC%E3%83%89200%E7%95%AA%E5%8F%B0%E3%80%80%E6%88%90%E5%8A%9F%E3%83%AC%E3%82%B9%E3%83%9D%E3%83%B3%E3%82%B9,-HTTP%E3%82%B9%E3%83%86%E3%83%BC%E3%82%BF%E3%82%B9%E3%82%B3%E3%83%BC%E3%83%89

Reply:
修正しました！